packages/api/.eslintrc.js

```diff
@@ -1,7 +1,18 @@
 /*global module*/
 module.exports = {
-  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended', 'prettier'],
-  parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"],
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint", "@metriport/eslint-rules"],
   root: true,
-};
\ No newline at end of file
+  rules: {
+    "@metriport/eslint-rules/no-named-arrow-functions": "warn",
```

Comment:
can we refer to the "recommended" config, so we pull all Metriport recommended settings?